app/controllers/books/new.js

```diff
@@ -0,0 +1,3 @@
+import EditController from './edit';
+
+export default EditController;
```

Comment:
Ember doesn't like when you re-export components, for controllers it may work but maybe it's nicer and clearer intention when you do `export default class NewBookController extends EditController {}`?